test/runtests.jl

```diff
@@ -1,4 +1,4 @@
 using TestItemRunner
 
 # Run all testitem tests in package
-@run_package_tests
+@run_package_tests filter=ti->( !(:experimental in ti.tags) )
```

Comment:
marking the bilinear density integrator test as flaky and filtering it out here. opening an issue